src/test/kotlin/no/nav/familie/ef/iverksett/brev/BrevFunctionsKtTest.kt

```diff
@@ -0,0 +1,39 @@
+package no.nav.familie.ef.iverksett.brev
+
+import no.nav.familie.ef.iverksett.lagIverksett
+import no.nav.familie.kontrakter.ef.felles.BehandlingType.FØRSTEGANGSBEHANDLING
+import no.nav.familie.kontrakter.ef.felles.BehandlingType.REVURDERING
+import no.nav.familie.kontrakter.ef.felles.BehandlingÅrsak.NYE_OPPLYSNINGER
+import no.nav.familie.kontrakter.ef.felles.BehandlingÅrsak.SØKNAD
+import no.nav.familie.kontrakter.ef.felles.Vedtaksresultat.AVSLÅTT
+import no.nav.familie.kontrakter.ef.felles.Vedtaksresultat.INNVILGET
+import no.nav.familie.kontrakter.ef.felles.Vedtaksresultat.OPPHØRT
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+
+internal class BrevFunctionsKtTest {
+
+    private val iverksettFørsteGangsbehandlingInnvilget =
```

Comment:
Det var mye oppsett for å teste type/vedtaksresultat :P Burde lagVedtakstekst egentlige kun fått inn type og vedtaksresultat for å være enklere å teste?